webpack/assets/javascripts/react_app/components/AutoComplete/index.js

```diff
@@ -42,7 +42,7 @@ class AutoComplete extends React.Component {
       return;
     }
     const instance = this._typeahead.current.getInstance();
-    switch (e.keyCode) {
+    switch (e.charCode) {
```

Comment:
good catch! can you please add a test :-)

Reply:
sure :)